conjur/cli.py

```diff
@@ -162,7 +162,7 @@ def run(self, *args):
                                                                                     '    conjur login -i admin \t\t\t'
```

Comment:
won't multilie string serve you better than this infinite string concatenation?

Reply:
as in `""""` instead?

Reply:
As in
```python
'''
multiline string
'''
```

Reply:
then it makes it sensitive to formatting which will make it ugly